lib/dor/models/identifiable.rb

```diff
@@ -210,10 +210,11 @@ def update_tag(old_tag, new_tag)
 
     def get_related_obj_display_title(related_obj, default_title)
       # desc_md_ds_title will be nil if related_obj is nil
-      desc_md_ds_title = if related_obj
-        desc_md_ds = related_obj.datastreams['descMetadata']
-        desc_md_ds ? desc_md_ds.title_info.main_title.first : nil
-      end
+      desc_md_ds_title =
+        if related_obj
+          desc_md_ds = related_obj.datastreams['descMetadata']
+          desc_md_ds ? desc_md_ds.title_info.main_title.first : nil
+        end
 
```

Comment:
This assignment is sub-optimal to begin with.  It could be simplified to include:
```ruby
return default_title unless related_obj
```

That removes the entire weird assignment conditional prompting this formatting issue.